src/svpcvp.cpp

```diff
@@ -155,7 +166,11 @@ static int shortestVectorEx(IntMatrix& b, IntVect& solCoord,
     Float ftmp1;
     bool result = evaluator->getMaxErrorAux(maxDist, true, ftmp1);
     FPLLL_CHECK(result, "shortestVector: cannot compute an initial bound");
-    maxDist.add(maxDist, ftmp1, GMP_RNDU);
+    if (flags & SVP_DUAL) {
+      maxDist.add(maxDist, ftmp1, GMP_RNDU);
```

Comment:
Actually, now that I think about it, I believe this is a relic from when I inverted the maxDist bound for dual enumeration later. Now I think the original line here would be the correct thing to do. Thank you for pointing that out. I'll update the pull request soon, where I'll include the changes from above as well.